scripts/syncup-s3.sh

```diff
@@ -63,6 +63,7 @@ sync_files=(
   scRNA-seq-advanced/data/ewing-sarcoma/annotations/ewing_sarcoma_sample_metadata.tsv
   scRNA-seq-advanced/data/rms/annotations/rms_sample_metadata.tsv
   scRNA-seq-advanced/data/reference/hs_mitochondrial_genes.tsv
+  scRNA-seq-advanced/data/wilms-tumor/processed/SCPCS000203/SCPCL000240_processed.rds
```

Comment:
We should not need to include the individual file here. The only time this is needed is if we don't want to upload the entire folder. Nothing nested in a directory above should be needed here.

Reply:
done in https://github.com/AlexsLemonade/training-modules/pull/818/commits/6422de75f601000d745d68d7f12d2dc87ded84c1